config/environments/development.rb

```diff
@@ -24,4 +24,9 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  # Better logging for docker-compose
+  config.logger = Logger.new(STDOUT)
+
+  # PrxAuth::Rails.middleware = false
+  # config.middleware.insert_before 'ActionDispatch::ParamsParser', 'Rack::PrxAuth', cert_location: 'http://id.prx.dev/api/v1/certs', issuer: 'id.prx.dev'
```

Comment:
Yeah, I usually comment this out in cms-dev as well, and just use id.prx.org.  Maybe we should make that our default there too.